utils/websocketJitter.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import random from 'lodash/random';
+
+import {WebSocketMessage} from '@mattermost/client';
+import {ActionFunc, DispatchFunc} from 'mattermost-redux/types/actions';
+
+const JITTER_RANGE = 2000; // 2 seconds
```

Comment:
added the experimental setting

Reply:
<img width="1003" alt="image" src="https://user-images.githubusercontent.com/17708702/216926659-8aa6abbb-41bd-4a71-bb1d-0f9891208cb2.png">

Reply:
>  I'm a bit worried about running into bizarre behaviour/race conditions if these events are handled out of order due to the jitter, 

I think creating a config setting may not be the right way to solve this. IMO trying to work around a potential bug by introducing a config setting can create more issues than intended. We should look into what sort of issues can arise due to this rather than setting a conservative number. A sleep is not a guaranteed time limit, and in busy systems, it can even take longer. So reducing the sleep time might make these bugs even harder to reproduce because the chances they happen will reduce, but they'll still happen nonetheless.

We should first confirm 100% that there are no issues with introducing jitter. If there are, we should fix them first, or just not use jitter in those cases.